### PR TITLE
[JENKINS-71479] Do not use SCSS lighten function directly to avoid invalid CSS

### PR DESCRIPTION
**Pull request by dwnusbaum**
See [JENKINS-71479](https://issues.jenkins.io/browse/JENKINS-71479).

https://github.com/jenkinsci/jenkins/pull/7850 in 2.402 migrated frontend styles from LESS to SCSS, but accidentally broke the generated CSS styles for `--btn-primary-bg-hover` and `--btn-primary-bg-active`. Before that PR, the generated styles in `styles.css` were as follows:

```
--btn-primary-bg:#063f61;
--btn-primary-bg-hover:#085685;
--btn-primary-bg-active:#0a649b;
``` 

After that PR, the generated styles are these:

```
--btn-primary-bg:#063f61;
--btn-primary-bg-hover:lighten(#063f61,7.5%);
--btn-primary-bg-active:lighten(#063f61,12%);
```

As far as I understand, the latter is invalid CSS, and in my case it seems to be treated as transparent, resulting in buttons disappearing on hover.

With this PR, `styles.css` again has the intended colors:

```
--btn-primary-bg:#063f61;
--btn-primary-bg-hover:#085685;
--btn-primary-bg-active:#0a649b;
```

### Testing done

I tested this change manually against a (proprietary) plugin with a button with these CSS classes: `yui-button yui-submit-button submit-button primary`. Before the change, the buttons become invisible on hover, and after the change, they show the lighter color on hover as intended.

### Proposed changelog entries

- Fix invalid CSS which caused some buttons to become invisible on hover

### Proposed upgrade guidelines

N/A

```[tasklist]
### Submitter checklist
- [x] The Jira issue, if it exists, is well-described.
- [x] The changelog entries and upgrade guidelines are appropriate for the audience affected by the change (users or developers, depending on the change) and are in the imperative mood (see [examples](https://github.com/jenkins-infra/jenkins.io/blob/master/content/_data/changelogs/weekly.yml)). Fill in the **Proposed upgrade guidelines** section only if there are breaking changes or changes that may require extra steps from users during upgrade.
- [x] There is automated testing or an explanation as to why this change has no tests.
- [ ] New public classes, fields, and methods are annotated with `@Restricted` or have `@since TODO` Javadocs, as appropriate.
- [ ] New deprecations are annotated with `@Deprecated(since = "TODO")` or `@Deprecated(forRemoval = true, since = "TODO")`, if applicable.
- [ ] New or substantially changed JavaScript is not defined inline and does not call `eval` to ease future introduction of Content Security Policy (CSP) directives (see [documentation](https://www.jenkins.io/doc/developer/security/csp/)).
- [ ] For dependency updates, there are links to external changelogs and, if possible, full differentials.
- [ ] For new APIs and extension points, there is a link to at least one consumer.
```

### Desired reviewers

@janfaracik 

Before the changes are marked as `ready-for-merge`:

```[tasklist]
### Maintainer checklist
- [ ] There are at least two (2) approvals for the pull request and no outstanding requests for change.
- [ ] Conversations in the pull request are over, or it is explicit that a reviewer is not blocking the change.
- [ ] Changelog entries in the pull request title and/or **Proposed changelog entries** are accurate, human-readable, and in the imperative mood.
- [ ] Proper changelog labels are set so that the changelog can be generated automatically.
- [ ] If the change needs additional upgrade steps from users, the `upgrade-guide-needed` label is set and there is a **Proposed upgrade guidelines** section in the pull request title (see [example](https://github.com/jenkinsci/jenkins/pull/4387)).
- [ ] If it would make sense to backport the change to LTS, a Jira issue must exist, be a _Bug_ or _Improvement_, and be labeled as `lts-candidate` to be considered (see [query](https://issues.jenkins.io/issues/?filter=12146)).
```

<a href="https://gitpod.io/#https://github.com/jenkinsci/jenkins/pull/8425"></a>

### CODE REVIEW

File: war/src/main/scss/abstracts/_theme.scss
Comment (dwnusbaum) [inline war/src/main/scss/abstracts/_theme.scss:120]: I am far from a SCSS expert, so IDK whether this is the _right_ way to do things, but it does fix the issue.

**Review by janfaracik — APPROVED** [review]

**Comment by dwnusbaum** [comment]
Ah I suspect that this is also the cause of [JENKINS-71479](https://issues.jenkins.io/browse/JENKINS-71479), reported by @NotMyFault. I'll double-check and update the PR title if so.

**Review by timja — APPROVED** [review]
/label ready-for-merge

----

This PR is now ready for merge, after ~24 hours, we will merge it if there's no negative feedback.

Thanks!

**Review by aneveux — APPROVED** [review]

**Pull request merged**
